service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/Element.java

```diff
@@ -10,7 +10,7 @@
 
 @Data
 @Builder
-@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
can remove

Reply:
infact, just revert to what was there previously

Reply:
👍